pontos/github/api/artifacts.py

```
@@ -26,7 +26,9 @@ class GitHubAsyncRESTArtifacts(GitHubAsyncREST):
     def _get_paged_artifacts(
         self, api, *, params: Optional[Params] = None
     ) -> AsyncIterator[Artifact]:
-        return self._get_paged_items(api, "artifacts", Artifact, params=params)
+        return self._get_paged_items(
+            api, "artifacts", Artifact, params=params  # type: ignore
```

Can you explain the requirement of `# type: ignore` here?

---

That is because `_get_paged_items` returns an `AsyncIterator[GithubModel]`. (`GithubModel` is the parent of `Artifact`). But since we are giving it an Artifact as parameter, i figured that we will get in fact an `AsyncIterator[Artifact]` and the code behaves as wanted. So i hesitated to change any typing and simply ignore this error. If you think this should be changed in any way i can do that for sure